scripts/main.py

```diff
@@ -164,4 +169,14 @@ def speeches(date_list, debug=True):
             continue
 
     except ValueError:
-        print("Invalid input. Please enter a number.")
+        print("Invalid input. Please enter a number.") """
+
+try:
+    process_dates = dates_to_process(seed_dates_path)
+    get_json(process_dates)
+    sittings(process_dates, debug=False)
+    attendance(process_dates, debug=False)
+    topics(process_dates, debug=False)
+    speeches(process_dates, debug=False)
+except Exception as e:
+    print(f"An error occurred: {e}")
```

Comment:
great, thanks! 

i'm thinking: do we want to have two files? one for debugging, and another which represents the daily run.

i would like to continue to having a file which i can use for debugging or executing manual runs.

Reply:
yes actually that's a good idea！